### PR TITLE
[DOCS-219, -227, -228, -260, -266] Findings

### PR DESCRIPTION
**Pull request by ana-dashuk-cobalt**
### Added

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
| Findings | https://deploy-preview-254--cobalt-docs.netlify.app/platform-deep-dive/pentests/findings/ | |
| Review and Take Action on Findings | https://deploy-preview-254--cobalt-docs.netlify.app/platform-deep-dive/pentests/findings/remediate-findings/ | I worked with Molly earlier to draft content for Free Retesting Duration |
| Finding States | https://deploy-preview-254--cobalt-docs.netlify.app/platform-deep-dive/pentests/findings/finding-states/ |  |

### Updated

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
| Severity Levels | https://deploy-preview-254--cobalt-docs.netlify.app/platform-deep-dive/pentests/findings/severity-levels/ | Moved page |

+ Minor updates to other articles.

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

### CODE REVIEW

File: content/en/Platform Deep Dive/Pentests/Findings/severity-levels.md [hunk 1]
```diff
@@ -23,16 +24,14 @@ Risk = Likelihood * Impact
...
 May include factors such as:
 
   - Skill required for an attacker to exploit a [vulnerability](../getting-started/glossary/#vulnerability)
   - Availability of documented exploits
   - Ease of exploiting the vulnerability
 
-<br>
-
-- Impact: Depends on the effect on technical and business operations. May include:
+- **Impact**: Depends on the effect on technical and business operations. May include:
```
Comment (mjang-cobalt) [inline content/en/Platform Deep Dive/Pentests/Findings/severity-levels.md:34]: Given the label in the UI, should we change this to "Business Impact"?
Screenshot 2022-12-12 at 6 38 48 AM

**Review by mjang-cobalt — CHANGES_REQUESTED** [review]
Good stuff. But at minimum, 

- We need some info to the Free Retesting Duration section, in the remediate-findings.md file. 
   - Brainstorming only: Maybe it's an opportunity to try out your subscription labels?
- The **Type of vulnerability** is not limited to the OWASP Top 10 list -- the links are limited to Web app vulnerabilities

And also -- what tool did you use to annotate the ViewFindings.png file? I'd like to learn!

File: content/en/Platform Deep Dive/Pentests/Findings/remediate-findings.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+1. On the **Findings** tab of the pentest page, filter **Pending Fix** findings.<br><br>
+![Filter Pending Fix findings](/deepdive/FilterPendingFixFindings.png "Filter Pending Fix findings")
+1. Select the desired finding.
+1. In the **State** list, select **Ready for Retest**.<br><br>
+![Submit a finding for retest](/deepdive/ReadyForRetestFinding.png "Submit a finding for retest")
+
+The pentester who posted the finding gets notified and retests the issue. When finished, they change the [state](/platform-deep-dive/pentests/findings/finding-states/) to:
+
+- Fixed, if they can't reproduce the issue.
+- Pending Fix, if the issue persists. Read the pentester's comment for details.
```
[the remark this answers is not in the document]
Comment (mjang-cobalt) [reply content/en/Platform Deep Dive/Pentests/Findings/remediate-findings.md:33]: I'm not going to worry about this -- as users don't select these as menu options.

File: content/en/Platform Deep Dive/Pentests/Findings/severity-levels.md (hunk 1 above)
Comment (mjang-cobalt) [reply content/en/Platform Deep Dive/Pentests/Findings/severity-levels.md:34]: I waffled on this one -- maybe the UI label is misleading?

File: content/en/Platform Deep Dive/Pentests/Findings/remediate-findings.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+### Free Retesting Duration
```
Comment (mjang-cobalt) [inline content/en/Platform Deep Dive/Pentests/Findings/remediate-findings.md:35]: While I think this looks accurate, it bothers me.

- I think this is the kind of detail that belongs in our contract. I'd rather not involve our lawyers in our product docs
- The PtaaS Tier webpage already states the free retesting period

I'm guessing these details are also "subject to negotiation," which could affect the credibility of this section.

File: content/en/Platform Deep Dive/Pentests/Findings/remediate-findings.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+The timeline for retesting starts after your pentest end date within an active contract. Mark your findings as **Ready for Retest** at least 10 days before your contract ends.
+
+- Free retesting is based on your {{% ptaas-tier %}}:
+  - Standard tier: 6 months
+  - Premium and Enterprise tiers: 12 months
+- Free retesting is only available within an active contract. Your retest end date will be either the duration of your purchased tier or 10 days before your contract end date.
+- When you start a pentest right before your contract expires, you may not qualify for retesting. If you add a new contract, we’ll update your retest end date based on the tier your pentest was planned in.
+
+{{%expand "See an example." %}}
+The Standard contract duration is from January 1, 2022 to December 31, 2022.
```
Comment (mjang-cobalt) [inline content/en/Platform Deep Dive/Pentests/Findings/remediate-findings.md:46]: Wait... where did you get info for "The Standard contract duration"? Is there such a thing?

Unless legal has come to us and asked us to add this info, 

I recommend replacing this section. We can keep the link to https://www.cobalt.io/pentest-pricing. I might even include:

"Details and deadlines for free retesting depends on your contract. For more information, consult your CSM (if you have one), or contact support@cobalt.io."

**Comment by mjang-cobalt** [comment]
@ana-dashuk-cobalt your proposed "Free Retesting Duration" section bothers me. If you'd like to chat with legal, I'll support you. But I'd rather not include such details in our product docs unless they asked us to do so.

**Comment by ana-dashuk-cobalt** [comment]
> @ana-dashuk-cobalt your proposed "Free Retesting Duration" section bothers me. If you'd like to chat with legal, I'll support you. But I'd rather not include such details in our product docs unless they asked us to do so.

Please see my comment in the Added section: https://github.com/cobalthq/cobalt-product-public-docs/pull/254#issue-1487420619
We worked with Molly on this copy, it's currently published in https://cobaltio.zendesk.com/hc/en-us/articles/4408839706132-Cobalt-PtaaS-Tiers-Frequently-Asked-Questions#h_01GJ3N77K4VDQNTDJFT5P0GB5K

File: content/en/Platform Deep Dive/Pentests/Findings/severity-levels.md (hunk 1 above)
Comment (ana-dashuk-cobalt) [reply content/en/Platform Deep Dive/Pentests/Findings/severity-levels.md:34]: @mjang-cobalt Let's stick to the wording that OWASP is using: https://owasp.org/www-community/OWASP_Risk_Rating_Methodology

Risk = Likelihood * Impact

File: content/en/Platform Deep Dive/Pentests/Findings/remediate-findings.md (hunk 4 above)
Comment (ana-dashuk-cobalt) [reply content/en/Platform Deep Dive/Pentests/Findings/remediate-findings.md:46]: @mjang-cobalt Please see this comment: https://github.com/cobalthq/cobalt-product-public-docs/pull/254#issuecomment-1347425095

File: content/en/Platform Deep Dive/Pentests/Findings/remediate-findings.md (hunk 3 above)
Comment (ana-dashuk-cobalt) [reply content/en/Platform Deep Dive/Pentests/Findings/remediate-findings.md:35]: @mjang-cobalt Please see my comment here: https://github.com/cobalthq/cobalt-product-public-docs/pull/254#issuecomment-1347425095

**Review by mjang-cobalt — APPROVED** [review]

**Pull request merged**